lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -45,6 +45,15 @@ def script
 
           install_framework()
           {
+            if [ -e "$1" ]; then
+              version=$(defaults read "${PODS_ROOT}/Target Support Files/$(basename -s .framework "$1")/Info.plist" CFBundleShortVersionString)
```

Comment:
Hello guys! Thank you for the feedbacks! Ok I did not take into account that scenario. So my suggestion will be; what if we compare the modified timestamp of the source folder with the framework folder created in the project's derived data folder?